docs/Reference/Plugin-API-Interfaces.md

```diff
@@ -0,0 +1,19 @@
+description: Plugin interfaces
+<!--- END of page meta data -->
+
+Hyperledger Besu provides API interfaces to allow users to [build plugins](../Concepts/Plugins.md) to
+extend Besu functionality.
+
+
+
+| Interface          | Description                                              |
+|--------------------|----------------------------------------------------------|
```

Comment:
I would split this out into two tables.  BesuContext and BesuPlugin provide the core plugin classes.  All the other classes are services that can be retrieved.

This list is missing some services
**BesuConfiguration** - provides the file system locations of Besu's storage
**MetricCategoryRegistry** - adds a new metrics category to the CLI
**MetricsSystem** - Allows you to register metrics with the prometheus endpoint
**PoAQueryService** - For Clique and IBFT networks lets you query the current state of consensus
**IbftQueryService** - Like PoAQueryService but with extra IBFT specific queries.

in addition to PicoCLIOptions, BesuEvents, and StorageService

Reply:
Done.